mon-pix/app/templates/profile.hbs

```diff
@@ -51,12 +51,6 @@
       </div>
     </div>
 
-    {{#unless model.isProfileV2}}
-      <div class="profile-panel__link-to-compte">
```

Comment:
penser à supprimer la classe css `profile-panel__link-to-compte`

Reply:
Bien vu